### PR TITLE
Chore: small User documentation improvements

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Chore: small grammar and spelling improvements

### CODE REVIEW

File: doc_user/UserManual.adoc [hunk 1]
```diff
@@ -167,7 +167,7 @@ NOTE: to apply the above *Exclusive Filters* one has to just type a __number__ a
 ** *Date*: “Date From” and “Date To” to select all visits occurring during the specified period
 ** *Age*: “Age From” and “Age To” to select patients by age
 ** *Type of patient*: Select visits by type of attendance: All / New / Re-Attendance
-** *User*: Select visits by doctor (USER column): only visits inserted by the logged user (me) or All
+** *User*: Select visits by a doctor (USER column): only visits inserted by the logged user (me) or All
```
Comment (mizzioisf) [inline doc_user/UserManual.adoc]: I'm not sure this is better, since the word "doctor" here is the search field...I might be wrong !

**Review by mizzioisf — COMMENTED** [review]
I left just one comment here also, very nice work, thank you !

File: doc_user/UserManual.adoc (hunk 1 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: I see the point you are making; reverted the change

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**